app/models/item.rb

```diff
@@ -1,5 +1,8 @@
 class Item < ActiveRecord::Base
-  validates :title, presence: true
+  validates :description, length: { in: 0..255, allow_nil: false }, presence: true
+  validates :title, presence: true, uniqueness: true, allow_blank: false
+  validates :price, :format => { :with => /\A\d+(?:\.\d{0,2})?\z/ },
+            :numericality => {:greater_than => 0}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.